profile/2018-12-01-profile.json

```diff
@@ -0,0 +1,623 @@
+                             "locations/CheckDnsNameAvailability",
+                             "locations/checkAcceleratedNetworkingSupport",
+                             "locations/effectiveResourceOwnership",
+                             "locations/operationResults",
+                             "locations/operations",
+                             "locations/supportedVirtualMachineSizes",
+                             "locations/setResourceOwnership",
+                             "locations/usages",
+                             "locations/validateResourceOwnership",
+                             "locations/virtualNetworkAvailableEndpointServices",
+                             "networkIntentPolicies",
+                             "networkInterfaces",
+                             "networkProfiles",
+                             "networkSecurityGroups",
+                             "networkWatchers",
+                             "networkWatchers/connectionMonitors",
+                             "networkWatchers/pingMeshes",
+                             "operations",
+                             "publicIPAddresses",
+                             "publicIPPrefixes",
+                             "routeFilters",
+                             "routeTables",
+                             "serviceEndpointPolicies",
+                             "trafficmanagerprofiles",
+                             "trafficManagerGeographicHierarchies",
+                             "virtualNetworkGateways",
+                             "virtualNetworks",
+                             "virtualNetworkTaps"
+                         ]
+        },
+
+        "microsoft.notificationHubs": {
+
+                         "2016-03-01": [
+                             "operationResults",
+                             "operations"
+                         ],
+                         "2017-04-01": [
+                             "checkNamespaceAvailability",
+                             "namespaces",
+                             "namespaces/notificationHubs"
+                         ]
+        },
+
+        "microsoft.policyInsights": {
+
+                         "2018-04-04": [
+                             "operations",
+                             "policyEvents",
+                             "policyStates"
+                         ]
+
+        },
+
+        "microsoft.powerBI": {
+
+                        "2016-01-29": [
+                            "locations",
+                            "locations/checkNameAvailability",
+                            "workspaceCollections"
+                        ]
+        },
+
+        "microsoft.powerBIDedicated": {
+                        "2017-10-01":[
+                            "capacities",
+                            "locations/checkNameAvailability",
+                            "locations/operationresults",
+                            "locations/operationstatuses"

```

Comment:
`locations/operationresults` and `locations/operationstatuses` endpoint does not exists in microsoft.powerBIDedicated `2017-10-01` see [here](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/powerbidedicated/resource-manager/Microsoft.PowerBIdedicated/stable/2017-10-01/powerbidedicated.json).